web_src/js/features/imagediff.js

```diff
@@ -110,29 +110,42 @@ export function initImageDiff() {
     const $imagesAfter = imageInfos[0].$images;
     const $imagesBefore = imageInfos[1].$images;
 
-    initSideBySide(createContext($imagesAfter[0], $imagesBefore[0]));
+    initSideBySide(this, createContext($imagesAfter[0], $imagesBefore[0]));
     if ($imagesAfter.length > 0 && $imagesBefore.length > 0) {
       initSwipe(createContext($imagesAfter[1], $imagesBefore[1]));
       initOverlay(createContext($imagesAfter[2], $imagesBefore[2]));
     }
 
-    $container.find('> .image-diff-tabs').removeClass('is-loading');
+    for (const el of this.querySelectorAll(':scope > .image-diff-tabs')) {
+      el.classList.remove('is-loading');
+    }
 
-    function initSideBySide(sizes) {
+    function initSideBySide(el, sizes) {
```

Comment:
```suggestion
    function initSideBySide(container, sizes) {
```

I think it's better to use `container` as the argument name, to match the meaning of the old code.

Reply:
Lint doesn't allow `const container = this;`

Reply:
I mean `function initSideBySide(container, sizes) `, but not `const container = this;`

Reply:
[`7e657f3` (#30140)](https://github.com/go-gitea/gitea/pull/30140/commits/7e657f33e268c25deb734f202a6dd6ac9a43feb8)